posts/models.py

```diff
@@ -1,17 +1,36 @@
 from django.db import models
 from users.models import User
+from multiselectfield import MultiSelectField
 # Create your models here.
 
 
+class Ingredients(models.Model):
+    name = models.CharField(max_length=120, blank=False)
+
+
 class Post(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    INGREDIENT_CHOICES = (
+        ('tomato', 'Tomato'),
+        ('eggplant', 'Eggplant'),
+        ('celery', 'Celery'),
+        ('egg', 'Egg'),
+        ('milk', 'Milk'),
+        ('fish', 'Fish'),
+        ('chicken', 'Chicken'),
+        ('oil', 'Oil')
+    )
+    # picked_ingredients
+    ingredients = models.ManyToManyField(Ingredients)
```

Comment:
You can add `related_name`. `related_name="posts"` and use `posts` to access post of ingredients -> `ingredient.posts` `ingredients.posts.all()`
https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.ManyToManyField.related_name